spec/models/profile_spec.rb

```diff
@@ -195,91 +195,58 @@
   end
 
   describe '#decrypt_pii' do
-    it 'decrypts PII' do
-      expect(profile.encrypted_pii).to be_nil
-
+    it 'decrypts the PII for users with a multi region ciphertext' do
       profile.encrypt_pii(pii, user.password)
 
+      expect(profile.encrypted_pii_multi_region).to_not be_nil
+
       decrypted_pii = profile.decrypt_pii(user.password)
 
       expect(decrypted_pii).to eq pii
     end
 
-    it 'fails if the encryption context from the uuid is incorrect' do
+    it 'decrypts the PII for users with only a single region ciphertext' do
       profile.encrypt_pii(pii, user.password)
+      profile.update!(encrypted_pii_multi_region: nil)
 
-      allow(profile.user).to receive(:uuid).and_return('a-different-uuid')
+      decrypted_pii = profile.decrypt_pii(user.password)
 
-      expect { profile.decrypt_pii(user.password) }.to raise_error(Encryption::EncryptionError)
+      expect(decrypted_pii).to eq pii
     end
 
-    context 'with aws_kms_multi_region_read_enabled enabled' do
-      before do
-        allow(IdentityConfig.store).to receive(:aws_kms_multi_region_read_enabled).and_return(true)
-      end
-
-      it 'decrypts the PII for users with a multi region ciphertext' do
-        profile.encrypt_pii(pii, user.password)
-
-        expect(profile.encrypted_pii_multi_region).to_not be_nil
-
-        decrypted_pii = profile.decrypt_pii(user.password)
-
-        expect(decrypted_pii).to eq pii
-      end
-
-      it 'decrypts the PII for users with only a single region ciphertext' do
-        profile.encrypt_pii(pii, user.password)
-        profile.update!(encrypted_pii_multi_region: nil)
+    it 'fails if the encryption context from the uuid is incorrect' do
+      profile.encrypt_pii(pii, user.password)
 
-        decrypted_pii = profile.decrypt_pii(user.password)
+      allow(profile.user).to receive(:uuid).and_return('a-different-uuid')
 
-        expect(decrypted_pii).to eq pii
-      end
+      expect { profile.decrypt_pii(user.password) }.to raise_error(Encryption::EncryptionError)
     end
   end
 
   describe '#recover_pii' do
-    it 'decrypts the encrypted_pii_recovery using a personal key' do
-      expect(profile.encrypted_pii_recovery).to be_nil
-
+    it 'decrypts the recovery PII for users with a multi region ciphertext' do
```

Comment:
For those of us who don't automatically know that `recover_pii` uses the personal key
```suggestion
    it 'decrypts the recovery PII for users with a multi region ciphertext personal key' do
```